test/models/observation_test.rb

```diff
@@ -95,10 +125,23 @@ def test_specimens
     assert(observations(:detailed_unknown).specimens.length > 0)
   end
 
+  def test_observer_takes_general_email_questions_from_viewer
+    user_a = users(:rolf) # rolf takes general email questions
```

Comment:
Got this note about
observation_test#test_observer_takes_general_email_questions_from_viewer
just after I re-opened the PR, and while Travis was running.
I will re-write the test and push it up, so there will be more msgs from
Travis.  May not happen until tonight Pacific Time.
Writing fixtures can be a royal pain, but there seems to be lots of
controversy about Fixtures vs. Factories. See e.g.:
http://brandonhilkert.com/blog/7-reasons-why-im-sticking-with-minitest-and-fixtures-in-rails/
https://whatdoitest.com/getting-friendly-with-fixtures
https://semaphoreci.com/blog/2014/01/14/rails-testing-antipatterns-fixtures-and-factories.html
https://www.google.com/search?q=rails+fixtures+versus+factories&rlz=1C5CHFA_enUS512US512&oq=rails+fixtures+versus+factories&aqs=chrome..69i57.8767j0j7&sourceid=chrome&es_sm=119&ie=UTF-8




On Sun, Oct 18, 2015 at 11:17 AM Nathan Wilson <notifications@github.com>
wrote:

> In test/models/observation_test.rb
> <https://github.com/MushroomObserver/mushroom-observer/pull/187#discussion_r42324028>
> :
>
> > @@ -95,10 +125,23 @@ def test_specimens
> >      assert(observations(:detailed_unknown).specimens.length > 0)
> >    end
> >
> > +  def test_observer_takes_general_email_questions_from_viewer
> > +    user_a = users(:rolf) # rolf takes general email questions
>
> Since we're using static fixtures, the way to do this is create a separate
> observation fixture for this test that has a relevant name. At some point
> we should consider moving to Factory Girl factories rather than static
> fixtures. They essentially let you do what you want to do here without undo
> database cost.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MushroomObserver/mushroom-observer/pull/187/files#r42324028>
> .
>